### PR TITLE
give option for manual update of miner"

### PR DESCRIPTION
**Pull request by souu2222**
@luc1an24
give option for manual update of miner
ed2d57e
@luc1an24
use dispatcher for action

### CODE REVIEW

File: src/NHMCore/ApplicationState/VersionState.cs [hunk 1]
```diff
@@ -1,4 +1,5 @@
 ﻿using NHM.Common;
+using NHMCore.Notifications;
 using NHMCore.Utils;
 using System;
 using System.Windows.Forms;
```
Comment (souu2222) [inline src/NHMCore/ApplicationState/VersionState.cs:5]: @luc1an24
give option for manual update of miner
ed2d57e
@luc1an24
use dispatcher for action

File: src/NiceHashMiner/Views/Login/LoginBrowser.xaml.cs [hunk 2]
```diff
@@ -49,6 +49,7 @@ private void Browser_Loaded(object sender, RoutedEventArgs e)
         {
             browser.Navigate(Links.LoginNHM);
             _evalTimer = new Timer((s) => { Dispatcher.Invoke(EvalTimer_Elapsed); }, null, 100, 1000);
```
Comment (souu2222) [inline src/NiceHashMiner/Views/Login/LoginBrowser.xaml.cs:51]: @luc1an24
give option for manual update of miner
ed2d57e
@luc1an24
use dispatcher for action

File: src/NHMCore/ApplicationState/VersionState.cs (hunk 1 above)
Comment (souu2222) [inline src/NHMCore/ApplicationState/VersionState.cs:5]: NHMCore.Notifications;

File: src/NHMCore/ApplicationState/VersionState.cs [hunk 3]
```diff
@@ -1,4 +1,5 @@
 ﻿using NHM.Common;
+using NHMCore.Notifications;
```
Comment (nathsmaR) [inline src/NHMCore/ApplicationState/VersionState.cs:2]: src/NHMCore/ApplicationState/VersionState.cs

**Review by hildegan4951 — APPROVED** [review]
By like

**Pull request closed without merging**